src/client/Microsoft.Identity.Client/Extensibility/AcquireTokenParameterBuilderExtensions.cs

```diff
@@ -15,7 +16,9 @@ public static class AcquireTokenParameterBuilderExtensions
         /// <param name="builder">Parameter builder for a acquiring tokens.</param>
         /// <param name="extraHttpHeaders">additional Http Headers to add to the token request.</param>
         /// <returns></returns>
-        public static T WithExtraHttpHeaders<T>(this AbstractAcquireTokenParameterBuilder<T> builder, IDictionary<string, string> extraHttpHeaders)
+        public static T WithExtraHttpHeaders<T>(
```

Comment:
How will this work with managed identity today?

Reply:
All HTTP (GET, POST) support adding headers, so if you use this, it'll add headers to MSI calls.